app/services/DateService.scala

```diff
@@ -42,9 +42,18 @@ class DateService @Inject()(clock: Clock) {
     }
   }
 
+  def lastDayOfCalendarQuarter: LocalDate = {
```

Comment:
Could we consolidate the naming in this class? We sometimes refer to "Period" and sometimes to "CalendarQuarter" - we should pick one and use it consistently (I don't mind which!)

Reply:
Yep agreed! Have now replaced "Period" with "Quarter"